src/classes/Model/Users/InvalidateToken.php

```diff
@@ -11,19 +11,21 @@ public function __construct(Database $database)
         $this->database = $database->dbObject;
     }
 
-    public function invalidate(int $userId)
+    public function invalidate(int $userId, string $token)
     {
-        $sql = "UPDATE
+        $sql = "DELETE FROM
```

Comment:
We can drop the `UAT_Used` column now as we are deleting them (and it wont be updated)

Reply:
Removed.